### ISSUE TITLE
Document expected interactions between near/far pointers and eye/gaze

### ISSUE THREAD

**Issue by wiwei** [microsoft/MixedRealityToolkit-Unity#4769]
A lot of folks who are using the MRTK are figuring out how near/far interactions and eye/gaze things play with each other through trial and error - it's literally impossible, however, for people to understand if things are happening are by design, or if there's a bug.

We need to author a doc explaining what the expected interactions are (whether this is in the MRTK itself, or be able to link to a docs.microsoft.com location that explains things)

**Comment by wiwei** [comment]
I spent some time thinking on this and chatting with folks on the team - this is actually a somewhat tricky area for a number of different reasons, and I'll go over each of the areas here.

It's difficult for me to write docs for this because these are still areas that we're actively doing research on (i.e. user interaction research to figure out what the optimal interaction models are).

There are a few things to call out like:

With the HL1 - style hand, the way it works is that you see the gaze cursor intially, but it disappears when you hand comes into place and starts interacting with things. This is primarily to keep consistency with the way that the HL1 OS works. It can be a bit confusing for folks who see the cursor disappear (it's possible to re-invoke this cursor by saying 'select' just like in the OS itself).

With the HL2 -style hand, eye and gaze is a bit weird - there's currently some code that prevents both eye gaze and hand ray interactions from both being active at the same time, partially because there hasn't been a lot of concrete recommendations about how to design good scenarios around them (i.e. if i'm looking at something maybe shooting a ray also is confusing because i feel like I need to focus on it with the ray). It's an area that is still being researched right now and I think that once folks figure this out a bit more, this should be published at that point.

Basically this is still an area that we're trying to figure out as well, and as a result it's tough to get some clear docs on the matter.
